react-app/src/components/Footer/index.tsx

```diff
@@ -101,6 +101,9 @@ export const Footer = ({ email, address, showNavLinks }: Props) => {
 };
 
 export const FAQFooter = () => {
+  const showEligibilityFields = useFeatureFlag("CHIP_SPA_SUBMISSION");
```

Comment:
Here the flag is named as `showEligibilityFields`, but in `ChipDetails`, it's called `useDetailsForm`. That's a bit confusing